src/components/our-vision-story-blocks.js

```diff
@@ -30,8 +30,10 @@ const OurVisionStoryBlocks = () => {
         <StoryBlock
           imageSource={node.image.childImageSharp.fluid}
           altText={node.altText && node.altText}
+          description={node.description && node.description}
           fullText={node.full_text && node.full_text}
-          imageRight={index % 2}
+          imageRight={index % 2 ? false : true}
```

Comment:
so this is a fun bit, for the blocks in this section they need to reverse order image and text wise so I flip the true and false here